crates/zkvm-interface/src/input.rs

```diff
@@ -7,7 +7,7 @@ use serde::Serialize;
 #[derive(Clone)]
 pub enum InputItem {
     /// A serializable object stored as a trait object
-    Object(Arc<Box<dyn ErasedSerialize + Send + Sync>>),
+    Object(Arc<dyn ErasedSerialize + Send + Sync>),
```

Comment:
This is a bit unrelated with this PR but with the previous one. We could remove this extra layer of indirection.